component/site/libraries/bootstrap.php

```diff
@@ -909,11 +909,20 @@ public static function endPanel()
 	 */
 	public static function loadCss($includeMainCss = true, $direction = 'ltr', $attribs = array())
 	{
+		$params = JComponentHelper::getParams('com_jevents');
 		// Load Bootstrap main CSS
 		if ($includeMainCss)
 		{
-			JHtml::_('stylesheet', 'com_jevents/bootstrap.css', $attribs, true);
-			JHtml::_('stylesheet', 'com_jevents/bootstrap-responsive.css', $attribs, true);
+			switch ($params->get("bootstrapcss", 1))
+			{
+				case 1:
+					JHtml::_('stylesheet', 'com_jevents/bootstrap.css', $attribs, true);
+					JHtml::_('stylesheet', 'com_jevents/bootstrap-responsive.css', $attribs, true);
+					break;
+				case 2:
+					JHtmlBootstrap::loadCss();
+					break;
+			}
 			//JHtml::_('stylesheet', 'com_jevents/jevbootstrap/bootstrap-extended.css', $attribs, true);
 		}
 
```

Comment:
Looks good but we need to change this code in jevents.php in the frontend and other places in the JEvents code  too 
`if (JComponentHelper::getParams(JEV_COM_COMPONENT)->get("bootstrapcss", 1)==1)
{
	// This version of bootstrap has maximum compatibility with JEvents due to enhanced namespacing
	JHTML::stylesheet("com_jevents/bootstrap.css", array(), true);
	// Responsive version of bootstrap with maximum compatibility with JEvents due to enhanced namespacing
	JHTML::stylesheet("com_jevents/bootstrap-responsive.css", array(), true);
}
else if (JComponentHelper::getParams(JEV_COM_COMPONENT)->get("bootstrapcss", 1)==2)
{
	JHtmlBootstrap::loadCss();
}
`
It would be a good bit of tidying up